rsa_test.go

```diff
@@ -155,6 +160,25 @@ func TestRSAKeyParsing(t *testing.T) {
 
 }
 
+func TestRSAParsePublicKeyFromPEM_PKCS1(t *testing.T) {
```

Comment:
Could we integrate this into the existing `TestRSAKeyParsing` function? We could then rework the existing test functions to use the table-driven test paradigm.